Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "typedmap"
+version = "0.1.0"
+authors = ["Konrad Gołuchowski <kodieg@gmail.com>"]
+edition = "2018"
+description = "A hash map (dash map). A type of value is defined by trait implemented on key type"
+license = "MIT OR Apache-2.0"
+keywords = ["hashmap", "type", "typed", "map", "dashmap", "anymap", "typemap"]
+categories = ["data-structures", "concurrency"]
+repository = "https://github.com/kodieg/typedmap/"
+homepage = "https://github.com/kodieg/typedmap/"
```

Comment:
AFAIK it's discouraged to duplicated repository as homepage. Maybe remove homepage = line?

Reply:
:+1: